src/llv8-inl.h

```diff
@@ -150,16 +156,16 @@ ACCESSOR(Map, InstanceDescriptors, map()->kInstanceDescriptorsOffset,
          HeapObject)
 
 inline int64_t Map::BitField3(Error& err) {
-  return LoadField(v8()->map()->kBitField3Offset, err) & 0xffffffff;
+  return v8()->LoadUnsigned(LeaField(v8()->map()->kBitField3Offset), 4, err);
 }
 
 inline int64_t Map::InObjectProperties(Error& err) {
   return LoadField(v8()->map()->kInObjectPropertiesOffset, err) & 0xff;
 }
 
 inline int64_t Map::InstanceSize(Error& err) {
-  return (LoadField(v8()->map()->kInstanceSizeOffset, err) & 0xff) *
-         v8()->common()->kPointerSize;
+  return v8()->LoadUnsigned(LeaField(v8()->map()->kInstanceSizeOffset), 1, err)
```

Comment:
Let's run `make format` against this. Putting operator on the next line is definitely not what our style allows at the moment.